### ISSUE TITLE
docker caching does not seem to work

### ISSUE THREAD

**Issue by KagamiChan** [MicrosoftDocs/azure-devops-docs#11226]
Hi, I'm using the docker@2 task for building and pushing images, after I added the cache for docker, I got the error saying the folder does not exist, which sounds to me docker did not save anything into that location

```
Resolving key:
 - docker  [string]
 - "Linux" [string]
 - cache   [string]
Resolved to: docker|"Linux"|cache
ApplicationInsightsTelemetrySender will correlate events with X-TFS-Session ###
Getting a pipeline cache artifact with one of the following fingerprints:
Fingerprint: `docker|"Linux"|cache`
There is a cache miss.
tar: /home/vsts/work/1/docker: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
```

is there any extra configuration required?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0d9dbf88-7e55-d427-145e-d36b04fb197f
* Version Independent ID: a3f30c00-817c-9c05-7a57-3f2bf618821e
* Content: [Docker task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/docker?view=azure-devops)
* Content Source: [docs/pipelines/tasks/build/docker.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/tasks/build/docker.md)
* Product: **devops**
* Technology: **devops-cicd-tasks**
* GitHub Login: @azooinmyluggage
* Microsoft Alias: **atulmal**

**Comment by TimShererWithAquent** [comment]
@KagamiChan - Thanks for your question. It looks like you're working through an issue with your scenario or implementation, rather than an issue with the documentation. Please consider these resources:

- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps Support Bot](https://azuredevopsvirtualagent.azurewebsites.net/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

**Comment by KagamiChan** [comment]
Hi @TimShererWithAquent, Thanks, looks like I chose a wrong page, let me rephrase and file another thread
